doc/source/cice_index.rst

```diff
@@ -729,6 +730,7 @@ section :ref:`tabnamelist`.
    "trestore", "restoring time scale", "days"
```

Comment:
Need to add tr_pond_sealvl just above here

Reply:
Fixed